release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -822,6 +822,61 @@ The Local Storage Operator now has the ability to:
 
 For more information, see xref:../storage/persistent_storage/persistent-storage-local.adoc#local-storage-discovery_persistent-storage-local[Automating discovery and provisioning for local storage devices].
 
+[id="ocp-4-6-registry"]
+=== Registry
+
+[id="ocp-4-6-pruner-tolerates-invalid-images"]
+==== Image pruner tolerates invalid images
+
+The image pruner now tolerates invalid image references by default on new installations of {product-title}, which allows pruning to continue even if it finds invalid images.
+
+[id="ocp-4-6-pruner-loglevel"]
+==== Change the image pruner's log level
+
+Cluster administrators can now configure `logLevel` in the Pruning Custom Resource to debug logs.
+
+//See xre:../applications/pruning-objects.adoc#pruning-images_pruning-objects[Automatically pruning images] for more information.
+
+[id="ocp-4-6-registry-azure-gov"]
+==== Image Registry supports Azure Government
+
+You can now set up and configure the Image Registry for Azure Government.
```

Comment:
Does `Image Registry` need to be capitalized? I noticed it is mostly lowercase in the docs unless referring to the Operator.

Reply:
Thank you - I was in Operator mindset. Updating now!